bundles/CoreBundle/config/services_workflow.yaml

```diff
@@ -73,3 +72,5 @@ services:
 
     Pimcore\Workflow\Dumper\StateMachineGraphvizDumper:
         public: true
+
+    Pimcore\Workflow\ActionsButtonService: ~
```

Comment:
Please remove since this has been moved. see https://github.com/pimcore/pimcore/blob/11.x/bundles/AdminBundle/config/services.yaml#L73

Reply:
thx - done in https://github.com/pimcore/pimcore/pull/14797/commits/6a8d1d16d8ef0e0c33f8474c8a03c110a64dad8c